src/config.rs

```diff
@@ -24,9 +24,13 @@ pub struct Config {
 	#[serde(alias = "LIBREDDIT_SFW_ONLY")]
 	pub(crate) sfw_only: Option<String>,
 
-	#[serde(rename = "REDLIB_DEFAULT_THEME")]
-	#[serde(alias = "LIBREDDIT_DEFAULT_THEME")]
-	pub(crate) default_theme: Option<String>,
```

Comment:
Is there any way we can preserve this behavior so someone with the `REDLIB_DEFAULT_THEME` env var set doesn't have to change it? Possibly by giving `default_theme_light` and `default_theme_dark` an alias of `REDLIB_DEFAULT_THEME` in order to make it the default across both?